app/assets/javascripts/osom-tables.js

```diff
@@ -150,14 +150,14 @@
 
   // Callback for successful ajax load of table data
   function table_load_success(container, new_content, url) {
-    var new_container = $(new_content);
-    container.html($(new_content).children());
```

Comment:
if the new_content looks like this:
```
<h5>
Recommended Job Seekers
<a title="" data-modal="true"><span class="icon icon-add"></span> </span></a>
</h5>
<div class="osom-table "><table> .... </table>
</div>
```
this line 154 would cause header repeat issue, in my fix I extract only the content inside the div `osom-table`